core/hash/comparison_spec.rb

```diff
@@ -0,0 +1,97 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+ruby_version_is "2.3" do
+  describe "Hash#<=" do
+    before :each do
+      @h1 = {a:1, b:2}
+      @h2 = {a:1, b:2, c:3}
+    end
+
+    it "expects h1 <= h1 to return true" do
```

Comment:
In general, it's better if the spec descriptions explain the behavior, for instance here it could be:
`returns true when comparing against itself` or `... against the same hash`.